frog/enhance-body.rkt

```diff
@@ -28,12 +29,12 @@
 (define (syntax-highlight xs)
   (for/list ([x xs])
     (match x
-      [(or `(pre ([class ,brush]) (code () ,(? string? texts) ...))
-           `(pre ([class ,brush]) ,(? string? texts) ...))
+      [(or `(pre ([class ,brush]) (code () ,texts ...))
+           `(pre ([class ,brush]) ,texts ...))
        (match brush
          [(pregexp "\\s*brush:\\s*(.+?)\\s*$" (list _ lang))
           `(div ([class ,(str "brush: " lang)])
-                ,@(pygmentize (apply string-append texts) lang))]
+                ,@(pygmentize (apply string-append (map xexpr->markdown texts)) lang))]
```

Comment:
To be clear, in my previous comment I meant should we use `xexpr->string` instead of `xexpr->markdown` _here_ -- in the argument to `map`?

Would that produce a valid string for Pygments?